en/qgc/video_streaming_wifi_broadcast.md

```diff
@@ -1,99 +1,118 @@
 
  - The transmitter sends its data regardless of any associated receivers. Thus there is no risk of sudden video stall due to the loss of association
- - The receiver receives video as long as it is in range of the transmitter. If it gets slowly out of range the video quality degrades but does not stall. Even if frames are erroneous they will be displayed instead of being rejected.
+
+ - The receiver receives video as long as it is in range of the transmitter. If it gets slowly out of range the video quality degrades but does not stall.
+
  - The traditional scheme “single broadcaster – multiple receivers” works out of the box. If bystanders want to watch the video stream with their devices they just have to “switch to the right channel”
+
  - Wifibroadcast allows you to use several low cost receivers in parallel and combine their data to increase probability of correct data reception. This so-called software diversity allows you to use identical receivers to improve relieability as well as complementary receivers (think of one receiver with an omnidirectional antenna covering 360° and several directional antennas for high distance all working in parallel)
+
  - Wifibroadcast uses Forward Error Correction to archive a high reliability at low bandwidth requirements. It is able to repair lost or corrupted packets at the receiver.
 
 
+
 ## Hardware modification.
-Alpha WUS051NH is a high power card and eats too much current while TX. If you power it from USB will reset port on Odroid C1/C0.
+Alpha WUS051NH is a high power card and eats too much current while TX. If you power it from USB will reset port on most ARM boards.
 So you need to connect it to 5V BEC directly. You can do this two ways:
 
  1. Make a custom usb cable.
- 2. Cut a 5V wire on odroid PCB near usb port and wire it to BEC.
+ 2. Cut a 5V wire on PCB near usb port and wire it to BEC.
     Also I suggest to add 470uF low ESR capacitor (like ESC has) between power and ground to filter voltage spikes.
 
 ## Software setup
-Download wifibroadcast [sources](https://github.com/svpcom/wifibroadcast).
 
-You need to patch kernel to:
- 
- 1. Enable TX rate lock. Use ``mac80211-radiotap-bitrate_mcs_rtscts.linux-4.4.patch``. Instead there are no way to specify data rate for injected radiotap packets.
- 2. Enable TX power lock. Use ``ez-wifibroadcast-1.4-kernel-4.4-patches.diff``. This will lock tx power to maximum supported by card.
- 3. Enable RX of frames with bad FSC (checksum). Use ``ez-wifibroadcast-1.4-kernel-4.4-patches.diff``. This is optional and don't use in current code.
+1. Install libpcap and libsodium development libs.
+2. Download wifibroadcast [sources](https://github.com/svpcom/wifibroadcast).
+3. [Patch](https://github.com/svpcom/wifibroadcast/wiki/Kernel-patches) your kernel. You only need to patch kernel on TX (except if you want to use WiFi channel which is disabled in your region by CRDA).
```

Comment:
Are there any instructions/links on how to do this patching? The link states what the patches are, but would be great to have starting point for users wanting to try this.